app/helper-entitlements.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
`helper-plugin-entitlements.plist` has one more property(`allow-unsigned-executable-memory`).
Not sure `helper` needs that also.

Reply:
According to the comment here https://chromium.googlesource.com/chromium/src/+/3fa9da4e877a, `com.apple.security.cs.allow-unsigned-executable-memory required to
  run Flash content and NaCl modules`. Not sure if widevine needs the same, but the approach with adding the other entitlement makes sense.

Reply:
@mkarolin Thanks for checking.
I think we don't need to add `allow-unsigned-executable-memory` for `helper`.
If this entitlement should be used for widevine, upstream also needs it.